src/ApiPort.VisualStudio/ApiPort.VisualStudio.csproj

```diff
@@ -31,7 +31,7 @@
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
     <Prefer32Bit>false</Prefer32Bit>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
```

Comment:
Sure. Are there plans to add a CI build for this? Otherwise, they may still pile up...